<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetDialog</class>
 <widget class="QDialog" name="SetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>871</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>参数设置</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>841</width>
     <height>321</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>苹方 常规</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QHeaderView::section
{
background-color: rgb(212, 211, 211);	
border-color: rgb(212, 211, 211);
border:0.4px solid;


}
QTableWidget::item::selected      
{



    color:rgb(255, 255, 255)   ;                    
    background: rgb(0, 158, 214);

       
}
</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>120</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>30</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>30</number>
   </attribute>
   <column>
    <property name="text">
     <string>方法片段名称</string>
    </property>
    <property name="font">
     <font>
      <family>苹方 常规</family>
      <pointsize>11</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>开始时间（min）</string>
    </property>
    <property name="font">
     <font>
      <family>苹方 常规</family>
      <pointsize>11</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>结束时间（min）</string>
    </property>
    <property name="font">
     <font>
      <family>苹方 常规</family>
      <pointsize>11</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>低质量端（m/z）</string>
    </property>
    <property name="font">
     <font>
      <family>苹方 常规</family>
      <pointsize>11</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>高质量端（m/z）</string>
    </property>
    <property name="font">
     <font>
      <family>苹方 常规</family>
      <pointsize>11</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>离子化模式</string>
    </property>
    <property name="font">
     <font>
      <family>苹方 常规</family>
      <pointsize>11</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>扫描模式</string>
    </property>
    <property name="font">
     <font>
      <family>苹方 常规</family>
      <pointsize>11</pointsize>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="Add">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>40</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>苹方 常规</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
/*按钮无任何操作时*/ 
/*背景色*/
	background-color: rgb(212, 211, 211);
	color: rgb(0, 0, 0);
border-radius:10px;
  
}
QPushButton:hover
{ /*鼠标悬停在按钮上时*/   

}
 QPushButton:pressed
{ /*按钮被按下时*/   
  background-color:rgb(0, 158, 214);
	color: rgb(255, 255, 255);
}
QPushButton:checked
{
 background-color:rgb(0, 158, 214);
	color: rgb(255, 255, 255);

}
</string>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>470</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>苹方 常规</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
/*按钮无任何操作时*/ 
/*背景色*/
	background-color: rgb(212, 211, 211);
	color: rgb(0, 0, 0);
border-radius:10px;
  
}
QPushButton:hover
{ /*鼠标悬停在按钮上时*/   

}
 QPushButton:pressed
{ /*按钮被按下时*/   
  background-color:rgb(0, 158, 214);
	color: rgb(255, 255, 255);
}
QPushButton:checked
{
 background-color:rgb(0, 158, 214);
	color: rgb(255, 255, 255);

}
</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>530</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>苹方 常规</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
/*按钮无任何操作时*/ 
/*背景色*/
	background-color: rgb(212, 211, 211);
	color: rgb(0, 0, 0);
border-radius:10px;
  
}
QPushButton:hover
{ /*鼠标悬停在按钮上时*/   

}
 QPushButton:pressed
{ /*按钮被按下时*/   
  background-color:rgb(0, 158, 214);
	color: rgb(255, 255, 255);
}
QPushButton:checked
{
 background-color:rgb(0, 158, 214);
	color: rgb(255, 255, 255);

}
</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Delete">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>90</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>苹方 常规</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
/*按钮无任何操作时*/ 
/*背景色*/
	background-color: rgb(212, 211, 211);
	color: rgb(0, 0, 0);
border-radius:10px;
  
}
QPushButton:hover
{ /*鼠标悬停在按钮上时*/   

}
 QPushButton:pressed
{ /*按钮被按下时*/   
  background-color:rgb(0, 158, 214);
	color: rgb(255, 255, 255);
}
QPushButton:checked
{
 background-color:rgb(0, 158, 214);
	color: rgb(255, 255, 255);

}
</string>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>861</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>苹方 常规</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>参数设置</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>60</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>苹方 常规</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>离子化时间</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>苹方 常规</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>扫描速率</string>
    </property>
   </widget>
   <widget class="QLabel" name="IonSave">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>苹方 常规</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>离子化存储水平</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PreIonTime">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>苹方 常规</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(255, 255, 255);
border:1px solid ;
border-radius:5px;
border-color: rgb(177, 177, 177);
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ScanRate">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>苹方 常规</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(255, 255, 255);
border:1px solid ;
border-radius:5px;
border-color: rgb(177, 177, 177);
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>苹方 常规</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(255, 255, 255);
border:1px solid ;
border-radius:5px;
border-color: rgb(177, 177, 177);
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>苹方 常规</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ms</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>苹方 常规</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>amu/s</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>苹方 常规</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>mz</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>410</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>苹方 常规</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
/*按钮无任何操作时*/ 
/*背景色*/
	background-color: rgb(212, 211, 211);
	color: rgb(0, 0, 0);
border-radius:10px;
  
}
QPushButton:hover
{ /*鼠标悬停在按钮上时*/   

}
 QPushButton:pressed
{ /*按钮被按下时*/   
  background-color:rgb(0, 158, 214);
	color: rgb(255, 255, 255);
}
QPushButton:checked
{
 background-color:rgb(0, 158, 214);
	color: rgb(255, 255, 255);

}
</string>
   </property>
   <property name="text">
    <string>保存方法</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OpenMenthod">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>300</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>苹方 常规</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
/*按钮无任何操作时*/ 
/*背景色*/
	background-color: rgb(212, 211, 211);
	color: rgb(0, 0, 0);
border-radius:10px;
  
}
QPushButton:hover
{ /*鼠标悬停在按钮上时*/   

}
 QPushButton:pressed
{ /*按钮被按下时*/   
  background-color:rgb(0, 158, 214);
	color: rgb(255, 255, 255);
}
QPushButton:checked
{
 background-color:rgb(0, 158, 214);
	color: rgb(255, 255, 255);

}
</string>
   </property>
   <property name="text">
    <string>打开方法</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>650</y>
     <width>861</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>苹方 常规</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>参数设置</string>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>60</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>苹方 常规</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>离子化时间</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>苹方 常规</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>扫描速率</string>
    </property>
   </widget>
   <widget class="QLabel" name="IonSave_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>苹方 常规</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>离子化存储水平</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PreIonTime_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>苹方 常规</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(255, 255, 255);
border:1px solid ;
border-radius:5px;
border-color: rgb(177, 177, 177);
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ScanRate_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>苹方 常规</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(255, 255, 255);
border:1px solid ;
border-radius:5px;
border-color: rgb(177, 177, 177);
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>苹方 常规</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(255, 255, 255);
border:1px solid ;
border-radius:5px;
border-color: rgb(177, 177, 177);
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>苹方 常规</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ms</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>苹方 常规</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>amu/s</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>苹方 常规</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>mz</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
